cli/core/accounts/api/account_api_service.py

```diff
@@ -0,0 +1,26 @@
+from cli.core.errors import wrap_mpt_api_error
+from cli.core.mpt.models import Token
+from mpt_api_client import MPTClient  # type: ignore
```

Comment:
To avoid adding this ignore every time it's imported, we can add it to the mypy.overrides property in the pyproject.toml file

```
[[tool.mypy.overrides]]
module = [
  "gunicorn.*",
  "pyfiglet.*",
]
ignore_missing_imports = true
```